### PR TITLE
new endpoint

### PR DESCRIPTION
**Pull request by paolocarta**

### CODE REVIEW

File: src/main/java/ch/jugs/jugsdemo/DemoApplication.java [hunk 1]
```diff
@@ -2,11 +2,19 @@
...
 @SpringBootApplication
+@RestController
 public class DemoApplication {
 
 	public static void main(String[] args) {
 		SpringApplication.run(DemoApplication.class, args);
 	}
+
+	@GetMapping
+	public String hello() {
```
Comment (paolocarta) [inline src/main/java/ch/jugs/jugsdemo/DemoApplication.java]: where is the test?

**Comment by bygui86** [comment]
forgive us Uncle Bob!!

File: src/main/java/ch/jugs/jugsdemo/DemoApplication.java (hunk 1 above)
Comment (paolocarta) [inline src/main/java/ch/jugs/jugsdemo/DemoApplication.java]: where is the test?

Comment (bygui86) [reply src/main/java/ch/jugs/jugsdemo/DemoApplication.java]: next PR :P

**Comment by bygui86** [comment]
:star: PR built and available in a preview environment **solidarchitectures-jugs-demo-pr-1** [here](http://jugs-demo.jx-solidarchitectures-jugs-demo-pr-1.35.197.239.119.nip.io)

**Pull request merged**